.github/workflows/split_monorepo_tagged.yaml

```diff
@@ -57,14 +57,14 @@ jobs:
 
             -
                 # Uses an action in the root directory
-                name: Monorepo Split of ${{ matrix.package }}
+                name: Monorepo Split of ${{ matrix.package.name }}
                 uses: symplify/github-action-monorepo-split@master
                 env:
                     GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
                 with:
-                    package-directory: 'packages/${{ matrix.package }}'
-                    split-repository-organization: 'symplify'
-                    split-repository-name: '${{ matrix.package }}'
+                    package-directory: '${{ matrix.package.path }}'
+                    split-repository-organization: '${{ matrix.package.organization }}'
```

Comment:
This suggest there are more organizations, which is not true.
It should be reverted to static single value.

Reply:
Different packages can perfectly have different organizations, even if you're not making use of it right now. For instance, you could deploy the migrify packages (that you recently moved under the symplify monorepo) under github.com/migrify instead of /symplify.

Even better, you could eventually integrate Rector + symplify in a single monorepo, if you ever decide to.

For my project, I certainly need it: I have plugin GraphQL API for WordPress (which is a final product) on a different repo than its components:

- https://github.com/getpop <= components
- https://github.com/GraphQLAPI <= product

But everything is hosted on the same monorepo: https://github.com/leoloso/PoP

Reply:
I've changed to get the organization per package directory, instead of per package. 

For that, I've added a new command `package-dir-organizations-json`:

```bash
bin/monorepo-builder package-dir-organizations-json
# Output:
# {"/Users/mac/GitRepos/GitHub/Libraries/symplify/symplify/packages":"symplify"}
```

So we can get the organization like this:

```yaml
echo "::set-output name=organizations::$(packages/monorepo-builder/bin/monorepo-builder package-dir-organizations-json)"
```

and:

```yaml
split-repository-organization: '${{ steps.output_data.outputs.organizations[matrix.package.directory] }}'
```